wpilibc/src/main/native/include/frc/Errors.h

```diff
@@ -142,6 +142,17 @@ namespace warn {
     }                                                                    \
   } while (0)
 
+/**
+ * Reports an error to the driver station (using HAL_SendError).
+ *
+ * @param[in]  format error message format
+ */
+#define FRC_ReportWarning(format, ...)                                   \
+  do {                                                                   \
+    ::frc::ReportError(warn::Warnings, __FILE__, __LINE__, __FUNCTION__, \
```

Comment:
WPIWarnings.mac has `warn::Warning`, not `warn::Warnings`. This PR could migrate instances of `FRC_ReportError(warn::Warning,`, but it still doesn't help with the other warning values.